src/ray/stats/metric_exporter.cc

```diff
@@ -20,7 +20,7 @@ namespace ray {
 namespace stats {
 
 template <>
-void MetricExporter::ExportToPoints(
+void MetricPointExporter::ExportToPoints(
```

Comment:
Shall we change its name in seprated PR if it's necessary. For protobuf-format-wise, we might keep the clean function.